.github/workflows/cypress.yml

```
@@ -0,0 +1,55 @@
+name: CI
+
+on:
```

Si tu veux tester tu peux essayer:
```suggestion
on:
  workflow_dispatch:
    inputs:
      cypress_host:
        description: host name
        type: string
        required: true
    secrets:
      CODECOV_TOKEN:
        description: Code coverrage token
        required: true
      SLACK_WEBHOOK:
        description: Webhook slack
        required: true
```